v1.1-beta/matrixx-digital-commerce/matrixx-sba-events/cnf-testsuite-results-20230807-145428-174.yml

```diff
@@ -0,0 +1,232 @@
+---
+name: cnf testsuite
+testsuite_version: v0.42.3
+status:
+command: /home/dwilmes/.mtx/konstruxx/working/tests/cnfTestsuite/cnf-testsuite cert
```

Comment:
Make this generic so it works in other environments besides your own system @deerowbear